integration-tests/macos.spec-int.js

```diff
@@ -43,6 +43,14 @@ test('insecure certificate', async ({ page }) => {
     await dash.hasInsecureTextDetail()
 })
 
+test('allowed first party requests', async ({ page }) => {
```

Comment:
I added a test case for the the new state, just for added confidence beyond the screenshots.